tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -160,6 +170,13 @@ export default {
     setVoteLevel() {
       cldrVote.setVoteLevelChanged(this.voteLevelChanged);
     },
+
+    setUnreadCount(n) {
+      this.unreadAnnouncementCount = n;
+      this.announcementsTitle = n
+        ? "You have " + n + " unread announcement(s)"
```

Comment:
someday we should use plurals…

Reply:
and MFWG2.0

Reply:
Or cheat and use:
"Unread announcements: " + n